moroScript.jsx

```diff
@@ -255,24 +279,50 @@
         },
         componentDidMount: function() {
           dictionary_data_promise.then(function(dictdata) {
-                this.setState({data: dictdata, loaded: true});
+                this.setState({data: dictdata, loaded: true}, function() {
+                  $(this.refs.right_half.getDOMNode()).sticky({});
+                }.bind(this));
               }.bind(this));
         },
         render: function() {
           if (this.state.loaded) {
-            //TODO: rendering all definitions is slow, so we only render 10000 for now. Add pagination before rendering all. @HSande
             return (
-             <div>
-                Dictionary({this.state.data.length}): <DictList data={this.state.data.slice(0,10000)}/>
+             <div className="ui grid">
+                <div className="eight wide column">
+                  Dictionary({this.state.data.length}): <DictList data={this.state.data}/>
+                </div>
+                <div className="eight wide column">
+                  <div ref='right_half' className="ui sticky">
+                    <RouteHandler data={this.state.data}/>
+                  </div>
+                </div>
               </div>
             );
           }
           return <div className="ui active dimmer">
-                  <div className="ui text loader">Loading</div>
-                 </div>
+                <div className="ui text loader">Loading</div>
+           </div>
         }
       });
 
+      // Dictionary view with concordance.
+      var ConcordanceView = React.createClass({
+        render: function() {
+          var text = "Definition for: " + this.props.params.morpheme + 
+            " is " + this.props.params.definition;
+          return <div className="ui segment">
+                {text}
```

Comment:
After {text} and before </div> would be the location to render the sentences that have the specified morpheme and definition.